pkg/addon/helper.go

```diff
@@ -83,6 +87,31 @@ func DisableAddon(ctx context.Context, cli client.Client, name string, config *r
 		}
 	}
 
+	if name == "velaux" {
```

Comment:
This is not a good implementation.

Reply:
This implementation is just same as the coding in /references/cli/addon.go.  
<img width="471" alt="1675416418906" src="https://user-images.githubusercontent.com/107997570/216563159-ff7875b0-55ce-4818-8dbf-2170b2775bc7.png">
Could you please give an better example that I can benefit from it.

Reply:
please refer to this issue: https://github.com/kubevela/kubevela/issues/5422